packages/perseus/src/types.ts

```diff
@@ -200,6 +200,9 @@ export type APIOptions = Readonly<{
         keypadHeight?: number,
         focusedElement?: HTMLElement,
     ) => unknown;
+    /**
+     * @deprecated - metadata is no longer used by the Group widget
+     */
```

Comment:
We're not able to just get rid of it?

Reply:
This is probably way too paranoid, but I want to avoid the situation where

- we delete `metadata` from the schema and parser.
- 5 years from now, someone decides it would be cool if `PerseusRenderer` had a `metadata` field.
- they get errors in production because legacy content already has `metadata`, and it's shaped totally differently from whatever new metadata format they came up with.

I guess the parser regression tests should stop them before they get to step 3, though?